src/main/java/de/tum/in/www1/artemis/service/LectureUnitProcessingService.java

```diff
@@ -96,7 +96,7 @@ private void removeBreakSlides(PDDocument document) {
             while (iterator.hasNext()) {
                 PDDocument currentPage = iterator.next();
                 String slideText = pdfTextStripper.getText(currentPage);
-                if (slideText.contains("Break")) {
+                if (slideText.contains("Break") || slideText.contains("Pause")) {
```

Comment:
`.contains()` is case-sensitive. You maybe need to adapt this with `.toLowerCase()`.

Reply:
As this check gets more complicated now: Maybe move it to a small helper method similar to `isOutlineSlide()`.
```suggestion
                if (isBreakSlide(slideText)) {
```

Reply:
@ge65cer  Yes i want it to case-sensitive. As far as i understand Break slides have the word Break or Pause with capital  letter.